specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+      "x-ms-client-name": "version",
+      "in": "header",
+      "required": true,
+      "type": "string",
+      "description": "Specifies the version of the operation to use for this request.",
+      "enum": [
+        "2018-10-10"
+      ]
+    },
+    "ClientRequestId": {
+      "name": "x-ms-client-request-id",
+      "x-ms-client-name": "requestId",
+      "in": "header",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the analytics logs when storage analytics logging is enabled."
+    },
+    "DataServiceVersionParameter": {
+      "name": "DataServiceVersion",
+      "in": "header",
+      "required": true,
+      "type": "string",
+      "description": "Specifies the data service version",
+      "x-ms-parameter-location": "method",
+      "enum": [
+        "3.0"
+      ]
+    },
+    "TableProperties": {
+      "name": "tableProperties",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/TableProperties"
+      },
+      "x-ms-parameter-location": "method",
+      "description": "The Table properties."
+    },
+    "formatParameter": {
+      "name": "$format",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "Specifies the media type for the response",
+      "x-ms-parameter-location": "method",
+      "x-ms-parameter-grouping": {
+        "name": "QueryOptions"
+      },
+      "x-ms-client-name": "Format",
+      "enum": [
+        "application/json;odata=nometadata",
+        "application/json;odata=minimalmetadata",
+        "application/json;odata=fullmetadata"
+      ]
+    },
+    "topParameter": {
+      "name": "$top",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "format": "int32",
+      "minimum": 0,
+      "description": "Maximum number of records to return.",
+      "x-ms-parameter-location": "method",
+      "x-ms-parameter-grouping": {
+        "name": "QueryOptions"
+      },
+      "x-ms-client-name": "Top"
+    },
+    "selectParameter": {
+      "name": "$select",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "Select expression using OData notation. Limits the columns on each record to just those requested, e.g. \"$select=PolicyAssignmentId, ResourceId\".",
+      "x-ms-parameter-location": "method",
+      "x-ms-parameter-grouping": {
+        "name": "QueryOptions"
+      },
+      "x-ms-client-name": "Select"
+    },
+    "filterParameter": {
+      "name": "$filter",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "OData filter expression.",
+      "x-ms-parameter-location": "method",
+      "x-ms-parameter-grouping": {
+        "name": "QueryOptions"
+      },
+      "x-ms-client-name": "Filter"
+    },
```

Comment:
Can we add parameters for the following:

"NextTableName" : query parameter on `Table_Query`
"NextPartitionKey"; query paramter on `Table_QueryEntitiesWithPartitionAndRowKey` and `Table_QueryEntities`
"NextRowKey": query paramter on `Table_QueryEntitiesWithPartitionAndRowKey` and `Table_QueryEntities`